From d0d72010f2c7b88d95c18b93f311dcab099e39a8 Mon Sep 17 00:00:00 2001
From: Daniel Charles <daniel.charles@intel.com>
Date: Thu, 11 Oct 2018 15:24:48 -0700
Subject: [PATCH] cmrtlib: remove cmrtlib from compilation

As cmrtlib is not a library that offers API/ABI, remove it from
compilation.

Driver code doesn't need cmrtlib to operate

Signed-off-by: Daniel Charles <daniel.charles@intel.com>
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 896d0b69..e323382a 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -42,7 +42,7 @@ endif()
 
 include(GNUInstallDirs)
 
-add_subdirectory(cmrtlib)
+#add_subdirectory(cmrtlib)
 
 add_subdirectory(media_driver)
 
@@ -52,7 +52,7 @@ endif()
 message(STATUS "LIBVA_DRIVERS_PATH = ${LIBVA_DRIVERS_PATH}")
 
 install (FILES ${CMAKE_CURRENT_BINARY_DIR}/media_driver/iHD_drv_video.so DESTINATION ${LIBVA_DRIVERS_PATH} COMPONENT media)
-install (FILES ${CMAKE_CURRENT_BINARY_DIR}/cmrtlib/linux/igfxcmrt64.so DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT media)
+#install (FILES ${CMAKE_CURRENT_BINARY_DIR}/cmrtlib/linux/igfxcmrt64.so DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT media)
 
 option (INSTALL_DRIVER_SYSCONF "Install driver system configuration file" OFF)
 if (INSTALL_DRIVER_SYSCONF)
-- 
2.17.1

